src/main/java/duke/task/TodoTask.java

```diff
@@ -0,0 +1,29 @@
+package duke.task;
+
+public class TodoTask extends Task {
+    static final String TASK_TYPE = "T";
```

Comment:
Here I think you can write private final String TASK_TYPE = "T".

Reply:
Noted with thanks!